cmds/main.go

```diff
@@ -134,14 +135,31 @@ func resultWriteCSV(list []lytics.TableWriter, name string) {
 }
 
 func resultWriteTable(cliCtx *cli.Context, list []lytics.TableWriter) {
-	table := termtables.CreateTable()
+	tableString := &strings.Builder{}
+	table := tablewriter.NewWriter(tableString)
 	for i, row := range list {
 		if i == 0 {
-			table.AddHeaders(row.Headers()...)
+			table.SetHeader(rowToString(row.Headers()))
 		}
-		table.AddRow(row.Row()...)
+		table.Append(rowToString(row.Row()))
 	}
-	fmt.Println(table.Render())
+	table.SetAutoFormatHeaders(false)
+	table.Render()
+	fmt.Println(tableString.String())
+}
+
+func rowToString(row []interface{}) []string {
+	res := []string{}
```

Comment:
[NIT] Since we're adding everything from `row` to the output you might as well make this slice that big to begin with:

```suggestion
	res := make([]string, 0, len(row))
```